src/mainWindow.cpp

```diff
@@ -513,6 +514,7 @@ void MWindow::openFileXML()
 
     if(!error_goal.isEmpty() or !error_routing.isEmpty() or !error_source.isEmpty() or !error_traffic.isEmpty())
     {
+        // Show error information when some files are locked
```

Comment:
```suggestion
        // Show error information when some files are missing
```

Ok then batter missing?

Reply:
Agreed!